README.md

```diff
@@ -16,6 +16,8 @@ The list of patches currently includes:
 
 _Note_: Apple CPU identifier must be `<0F01>` for 8 core CPUs or higher and `<0601>` for 1, 2, 4, or 6 cores. This is the default in OpenCore for non-natively supported CPUs.
 
+_Important Note_: The key `PlatformInfo > Generic > ProcessorType` is stored as a Number (Decimal), not Data (Hex). You cannot change this key to data, as OpenCore will break. In order to do the above, you will need to convert your desired ProcessorType from Hex to Decimal first, and put that in the ProcessorType key.
```

Comment:
This feels wrongly worded. OpenCore will not break, it is your configuration that is broken when you enter invalid data. Also, Xcode does allow entering 0x010F into numeric fields with automatic conversion, so I do not see a problem here.

Reply:
Hmm, yeah I see how that could be misinterpreted. Would "You cannot change this key to data, as OpenCore will reject the configuration on next boot." be better?
On your second point, while Xcode may automatically convert it, ProperTree does not. Perhaps that's something I ask CorpNewt about, but I don't think this issue is resolved by just telling people to use Xcode or not telling them at all.
The whole reason for this issue to exist is because it took me a while to figure it out, so wanted to add a note of clarification for the next person who is also stuck.

Reply:
Oh. Yeah that would probably work better.

Reply:
I talked to CorpNewt about this and he said that ProperTree DOES convert hex to dec automatically, but only if you write the hex as 0xFFFF, not <FFFF> as displayed in the readme. He added that he wasn't sure why its written as <FFFF> in this README and in the file included with the OC Installation Guide.
Given this, I think the straight forward solution for now is to just change <0601> to 0x0601 in the readme.

Reply:
0x0601 != <0601>. It must be 0x0106, due to endianness.

Reply:
Both Xcode & ProperTree will convert 0x0106 to 262 instead of the desired 1537. 
![image](https://user-images.githubusercontent.com/48845980/126031196-fc318dd9-ec53-445e-8dc4-8d054ac74fe5.png)

Reply:
Interesting, then it sounds like a mistake in the README. Ok, I guess we should change it to 0x601 then.